app/controllers/purchases_controller.rb

```diff
@@ -77,20 +77,31 @@ def destroy
 
   private
 
-  def clean_purchase_amount
+  def clean_purchase_amount_in_cents
     return nil unless params[:purchase][:amount_spent_in_cents]
 
     params[:purchase][:amount_spent_in_cents] = params[:purchase][:amount_spent_in_cents].gsub(/[$,.]/, "")
   end
 
+  def clean_purchase_amount_in_dollars
+    return nil unless params[:purchase][:amount_spent_in_dollars]
+
+    params[:purchase][:amount_spent_in_cents] = params[:purchase][:amount_spent_in_dollars].gsub(/[$,]/, "").to_d * 100
+  end
+
+  def clean_amount_spent_in_dollars
```

Comment:
I don't think this ended up being used anywhere - can you remove it?

Reply:
nice catch! I forgot to remove it.